### PR TITLE
Update version of gradle-git-properties in doc

### PR DESCRIPTION
**Pull request by rajadilipkolli**
upgrading to latest version solves the issue with newer gradle version.

This should be done only in 2.0.x+ branch as this plugin now needs[ java 8+](https://github.com/n0mer/gradle-git-properties)

### CODE REVIEW

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc [hunk 1]
```diff
@@ -2564,7 +2564,7 @@ plugin, as shown in the following example:
 [source,groovy,indent=0]
 ----
 	plugins {
-		id "com.gorylenko.gradle-git-properties" version "1.4.17"
+		id "com.gorylenko.gradle-git-properties" version "1.5.0"
```
Comment (snicoll) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc]: The latest release seems to be `1.4.21`. The [Gradle portal](https://plugins.gradle.org/plugin/com.gorylenko.gradle-git-properties) provides the same info. Where is that `1.5.0` coming from?

Comment (rajadilipkolli) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc]: @snicoll  When I check in gradle portal it is 1.4.21 but from the original developer, it is 1.5.0 . hence changed to 1.5.0
Reference : https://github.com/n0mer/gradle-git-properties

Comment (wilkinsona) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc]: As far as I can tell, the README in that repository is incorrect. Version 1.5.0 has yet to be released. @rajadileepkolli Can you please update this PR accordingly? We'd also greatly appreciate it if you did some due diligence and checked that your proposed changes will work before submitting them.

Comment (snicoll) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc]: It's a version hardcoded in a README. it's not on the gradle portal, nor anywhere else as far as I can see. I've updated the doc to `1.4.21`.

Comment (rajadilipkolli) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc]: @wilkinsona , My bad it's my mistake, Initially I have tested locally with 1.4.21 and raised pull request with it only when providing the link I have misread and changed the value to 1.5.0.

I will take care that this won't happens again.

**Pull request closed without merging**